Switch Key Vault sub resources to use Key Vault ID instead of URL (#2820)
terraform-providers/terraform-provider-azurerm: website/docs/r/key_vault_certificate.html.markdown

* `name` - (Required) Specifies the name of the Key Vault Certificate. Changing this forces a new resource to be created.

> * `vault_id` - (Required) Specifies the ID of the Key Vault instance the certificate resides in, available from the `azurerm_key_vault.id` property.

Reviewer: would it make sense to make this consistent with the other ID fields?

```suggestion
* `vault_id` - (Required) The ID of the Key Vault where the Certificate should be created.
```
